python/vmaf/core/train_test_model.py

```diff
@@ -740,7 +740,7 @@ def _delete(filename, **more):
             os.remove(filename)
 
     @classmethod
-    def get_xs_from_results(cls, results, indexs=None, aggregate=True):
+    def get_xs_from_results(cls, results, indexs=None, aggregate=True, features=None):
```

Comment:
Can you also update the signature of the derived class `NiqeTrainTestModel` under niqe_train_test_model.py?